service/src/main/resources/db/migration/V176__evaka_user_income_updated_by.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE income
+    ADD COLUMN updated_by_new uuid;
+
+UPDATE income
+SET updated_by_new = COALESCE(
+    (
+        SELECT id
+        FROM evaka_user
+        WHERE evaka_user.id = income.updated_by
```

Comment:
Changed this from `evaka_user.employee_id` to `evaka_user.id` because `updated_by` can contain person ids.